### PR TITLE
Chetan

### PR DESCRIPTION
**Pull request by chetanJTH**
appium server changes

### CODE REVIEW

File: src/main/java/UITestFramework/CreateSession.java [hunk 1]
```diff
@@ -222,19 +220,19 @@ public void startAppiumServer(String os) throws ExecuteException, IOException, I
...
-			command.addArgument("127.0.0.1");  
-			command.addArgument("--port", false);  
-			command.addArgument("4723");  
-			command.addArgument("--full-reset", false);  
-			DefaultExecuteResultHandler resultHandler = new DefaultExecuteResultHandler();  
-			DefaultExecutor executor = new DefaultExecutor();  
-			executor.setExitValue(1);  
-			executor.execute(command, resultHandler);  
-			Thread.sleep(5000);  
+			String appiumJSPath = "/Users/mac/.npm/lib/node_modules/appium/build/lib/main.js";
```
Comment (shankybnl) [inline src/main/java/UITestFramework/CreateSession.java:223]: Is the "appiumJSPath" property mandatory? I can start the server without it on my machine. Can you try on your end if it runs without this property?

File: src/main/java/UITestFramework/CreateSession.java [hunk 2]
```diff
@@ -222,19 +220,19 @@ public void startAppiumServer(String os) throws ExecuteException, IOException, I
...
+			builder = new AppiumServiceBuilder()
+					.withAppiumJS(new File(appiumJSPath))
+//                .usingDriverExecutable(new File(Appium_Node_Path))
+					.withIPAddress("0.0.0.0")
```
Comment (shankybnl) [inline src/main/java/UITestFramework/CreateSession.java:227]: `.withIPAddress("0.0.0.0")` is not required. It is default IP address

**Review by shankybnl — CHANGES_REQUESTED** [review]
Kindly look into the review comments. If you agree with them then make the same changes for windows appium start code as well. Thank you.

File: src/main/java/UITestFramework/CreateSession.java (hunk 1 above)
Comment (chetanJTH) [reply src/main/java/UITestFramework/CreateSession.java:223]: No, it's not required. I had checked with my code.

File: src/main/java/UITestFramework/CreateSession.java (hunk 2 above)
Comment (chetanJTH) [reply src/main/java/UITestFramework/CreateSession.java:227]: This line is also not required.

**Comment by shankybnl** [comment]
@chetanJTH - I have cherry-picked a commit from this pull request as there are multiple files that need not be merged. Changes are merged in this [commit](https://github.com/shankybnl/MobileAutomationFramework/commit/cdb0d5be5c4c595576c8786349ed2057cc004f5d). 

Thank you very much for your contribution :)

**Pull request closed without merging**